lib/Controllers/create.js

```diff
@@ -30,6 +57,9 @@ Create.prototype.write = function(req, res, context) {
         res.header('Location', location);
       }
 
+      // Reload for that all field are present
+      return instance.reload();
```

Comment:
We need always reload for that all fields are present. With include or not.

With reload:
```
{
"id": 10,
"title": "test",
"description": null,
"deadline": null,
"createdAt": "2015-06-11T13:14:29.889Z",
"updatedAt": "2015-06-11T13:14:29.889Z",
"projectId": null,
"project": null
}
```
Without:
```
{
"id": 6,
"title": "test",
"updatedAt": "2015-06-11T13:15:20.241Z",
"createdAt": "2015-06-11T13:15:20.241Z"
}
```